src/index.ts

```diff
@@ -27,10 +27,10 @@ export {
 export { default as Canvas, type CanvasProps } from "./Canvas";
 export { default as CanvasItem, type CanvasItemProps } from "./CanvasItem";
 export { default as EditLabelDialog } from "./EditLabelDialog";
-export { default as RoadMarking, type RoadMarkingProps } from "./RoadMarking";
-export { default as RoadSurface, type RoadSurfaceProps } from "./RoadSurface";
-export { default as TrafficSign, type TrafficSignProps } from "./TrafficSign";
-export { default as VehiclePath, type VehiclePathProps } from "./VehiclePath";
+// export { default as RoadMarking, type RoadMarkingProps } from "./RoadMarking";
+// export { default as RoadSurface, type RoadSurfaceProps } from "./RoadSurface";
+// export { default as TrafficSign, type TrafficSignProps } from "./TrafficSign";
+// export { default as VehiclePath, type VehiclePathProps } from "./VehiclePath";
```

Comment:
We should also comment out TrafficLight as its part of the same feature set. (Was mentioned in ticket comment)

Reply:
Can we also group the commented exports together at the bottom of the file and leave a comment as to why they are commented out. Leave a link to the bug ticket in YouTrack so can we can trace the detail when we come back and reinstate these in the future.